assets/sass/components/settings/_googlesitekit-settings-module.scss

```diff
@@ -272,7 +272,7 @@
 			.googlesitekit-settings-module__title {
 				width: min-content;
 
-				@media (min-width: $bp-wpAdminBarTablet) {
+				@media (min-width: $width-desktop) {
```

Comment:
Excellent spot @techanvil! There was a little mistake with the `min-width` there and have pushed a fix for it.

> If I've understood it correctly the intent is to avoid wrapping this "Thank with Google" text across multiple lines - have I got that right?

We actually decided to let the text wrap across multiple lines whenever possible to avoid the overlap.

> Incidentally while looking into this, I noticed a problem where text can overlap with the badge(s) at various viewport widths.

I pushed a fix for this and took a little different approach to improve this as much as possible. This includes wrapping multiple badges into two lines and aligning the status text to the right.

Reply:
Thanks @nfmohit, looks much better now!